### DISCUSSION TITLE
Trying to figure out my atomstack

### DISCUSSION THREAD

**Discussion by nitro912gr** [arkypita/LaserGRBL#1268] [Q&A]
Hello and happy new year! I have a lot of questions so please have patience 🥺 there is no TL/DR  I really need to get to the bottom of this to make sure I don't have a problematic laser module and contact support for the possibility of replacement.

I try to understand how my atomstack works.
First of all what is the speed limit of the machine? I tried a test with laserGRBL and then with lightburn and for some reason the same settings at the second was going way faster, to the point I stopped the test because I was afraid I'm pushing it. I'm still trying to figure out what was different but lightburn is too complicated and need some study, so maybe I missed some something.

Also how do we know what is the 100% power? I mean I select 1000 in the settings but does the laser module understand to go full power? Does it change anything if I select some preset from the ortur? 

Next is that I try to do some cutting with my atomstack A5 but I need way more passes than I see in videos from others.
To the point that I think that maybe my laser module is not as advertised.
I removed the side panels (I use the glasses all the time anyway, those panels doesn't cover every possible angle that you may look at the machine so I don't risk it) and added a fan at the side to help the fumes go away).

I used the hardwood cut setting (by the way the passes need to be added manually after I pick the setting or I have some bug?) that goes for the 400mm speed and 100%power and 15 passes and is supposed to cut through hardwood at 6mm, but I wasn't able to cut 3,5mm plywood and had to rise the passes to 25. Is it possible my plywood is harder than plywood?

Also the resulted cut was way tooooooo burned on the sides, if I add air assist I will get cleaner cuts?

At first I though the problem is that I don't have air assist installed but later on a dude in youtube cut without air assist a 4mm plywood with 600 speed, 60 power and 9 passes. I never managed even adding some light burn on the other side of the plywood 3.5mm that I have with those settings (he used lightburn, is it possible that is giving more power to the module somehow?)

I tried cutting the provided 2mm plywood that was in the laser package and it cut it with 600 speed 60power and 4 passes. Is this normal?

The only thing I cut with ease was 2.5mm balsa wood that took me 1 pass with 600 speed and 100% power. But this is really expensive wood to work with.

Is it possible that my 3.5 plywood is just too hard? But harder than 6mm hardwood? I got some hardwood to test today (then again every hardwood tree have different densities...)

Are there special plywoods for laser cutting? I strangled to find a shop to have 2mm plywood here in Greece and I ordered a couple of samples to test, the guy said he order those thin playwoods for lasercutting.

Thanks for your patience :)

**Comment by arkypita** [comment]
Any PC program talk to the machine in a standard language called g-code, that is the same for LaserGRBL and Lightburn.
The only difference between the two is which G-Codes are generated, that could be different.

However, the speed is set via gcode "F" command, the power is set via "S" command, and in LaserGRBL it is the one you set in "target image size and laser option" form. http://lasergrbl.com/usage/raster-image-import/target-image-size-and-laser-options/

LaserGRBL will command these values, but it is possible that the machine run slower because the grbl configuration (grbl is the firmware that run on the machine) has limits on speed and acceleration. Read http://lasergrbl.com/faq/

The same way, is it also possible that the machine use a lower power, because if you use "M4 - Dynamic power mode" as laser mode, the power is scaled down to the current speed, so if the machine is running slower because of limit, also the laser is less powerfull.

Again, read faq, configuration, usage section of LaserGRBL.com website.

In the upper-right part of the screen, in LaserGRBL, you can read the real time value reported by the controller

![image](https://user-images.githubusercontent.com/8782035/104699442-03856100-5713-11eb-942a-07f73e85c381.png)

**Comment by arkypita** [comment]
Keep in mind that machines usually come with speed and acceleration limits that have been determined by the manufacturer to ensure accurate positioning. If you decide to play with the configuration parameters of the machine, first save a copy of the original ones.

**Comment by arkypita** [comment]
> I tried cutting the provided 2mm plywood that was in the laser package and it cut it with 600 speed 60power and 4 passes. Is this normal?

I don't know. Why you dont' contact atomstack? http://atomstack3d.com

> Are there special plywoods for laser cutting?

Not all the plywood are the same. Different wood, different color, different density, different glues.
Yes, there are special plywood for laser cutting. Ask google.

**Reply by nitro912gr** [reply]
thank you for the replies, I'm reading through the FAQ now, I will contact them but community help is always faster and sometimes better :)

you are overall a great help and your videos explain a lot in youtube

**Comment by DtyRobert** [comment]
Answer for those who encounter this problem: all these things happen to him because you do not focus correctly! The laser must be 2mm from the material!

**Reply by nitro912gr** [reply]
Hello, actually it was the material I was using. I used 3.5mm plywood with strong and hard adhesive between it's layers.
After a lot of digging around in the local market I managed to find that one wood specialty shop that is trading plywood made with softer glues specifically for lasercutting.
I ordered some samples at 2 and 3mm and it cut it nicely.

For the focus I use that small black plexiglass provided with the machine, isn't it right?
